doc/documentation.adoc

```diff
@@ -6,7 +6,7 @@
 [source,javascript]
 ----
 gulp.pipe(asciidoctor({
-  header_footer: false,
+  header_footer: true,
```

Comment:
We should probable use the new terminology in the example to encourage users to use it:

```suggestion
  standalone: true,
```

Reply:
Should standalone = true, header_footer = true  be the default?

Reply:
The default value in the Asciidoctor CLI is true: https://github.com/asciidoctor/asciidoctor/blob/11aaeabed5935cdb7e65b65eb44b44210f039b87/lib/asciidoctor/cli/options.rb#L15

So I would say yes.

Reply:
Then it is probably also better to strip header_footer and rely on standalone. I will fix that.

- if only header_footer is set => standalone = value of header_footer
- if both are set, standalone = value of standalone and header_footer will be stripped